include/z64subs.h

```diff
@@ -97,7 +97,7 @@ typedef struct ActorPathing {
     /* 0x68 */ ActorPathingUpdateFunc setNextPointFunc; // Return true if should compute and update again
 } ActorPathing; // size = 0x6C
 
-struct EnDoor* SubS_FindDoor(struct PlayState* play, s32 switchFlag);
+struct EnDoor* SubS_FindDoorSchedule(struct PlayState* play, s32 schType);
```

Comment:
I keep coming back to this, but I kind of want to to do `FindScheduleDoor` or maybe even `FindScheduledDoor`. Right now the name makes me think it is going to find the schedule for the door (i.e. gets the script).

Reply:
`FindScheduleDoor` sounds fine, I'll change it